h/services/list_organizations.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h import models
+
+
+class ListOrganizationsService(object):
+
+    """
+    A service for providing a list of organizations.
+    """
+
+    def __init__(self, session, authority):
+        """
+        Create a new list_organizations service.
+
+        :param _session: the SQLAlchemy session object
+        :param _authority: the authority to use as a default
+        """
+        self._session = session
+        self._authority = authority
+
+    def organizations(self, authority=None):
+        """
+        Return a list of organizations filtered on authority.
+        """
+        return (self._session.query(models.Organization)
+                    .filter_by(authority=authority or self._authority)
```

Comment:
I like concise code, but perhaps this would be easier to read if the `authority` assignment were outside of the `filter_by` invocation?

Reply:
I'll break it out.